packages/ocp-browser/ocp-browser.1.0.0/opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+maintainer: "ligand@free.fr"
+authors: ["Francois BERENGER"]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
```

Comment:
It's better to leave out the dev-repo than put an incorrect one in, since pinning will fail due to the lack of an `opam` file.